src/main/java/com/agido/logback/elasticsearch/AbstractElasticsearchAppender.java

```
@@ -167,4 +168,8 @@ public void setObjectSerialization(boolean objectSerialization) {
     public void setAutoStackTraceLevel(String level) {
         settings.setAutoStackTraceLevel(level);
     }
+
+    public void setOperation(String operation) {
+        settings.setOperation( Operation.of( operation ).orElse( Operation.CREATE ) );
```

it is fine for me as it is, but could be unexpected behaviour: `<operation>foo</operation>`  (or any other valid value supported in a future version) would also silently use `CREATE` if it is not known in the current version.
the default value for the "not specified" case is defined in Settings.java, but if you explicitly set it to an unknown value, it could be better to at least output a warning, or maybe just throw an exception (i do not know the exact logback sematics here regarding invalid/unexpected configuration).

---

I am fine with throwing an exception for the "not specified" cases, I'll check the current valid values for op_type and check the logback conventions in case of invalid configuration options